src/pages/AddTasksForm.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { useState } from "react";
+import { useHistory } from "react-router-dom";
+import { Form, Label, Input, SubmitInput } from "../styled-components/form";
+import { postData, paramsUrl } from "../utils/utils";
+import { MainCont } from "../styled-components/reusables";
+import NavBar from "../components/NavBar";
+
+export default function AddTasksForm() {
+	const [taskname, setTaskName] = useState("");
+	const [description, setDescription] = useState("");
+
+	let history = useHistory();
+
+	const url = paramsUrl(
+		`http://localhost:4567/create-tasks`,
```

Comment:
`localhost:4567` is repeated a few times in the code - have you thought about DRYing it? how?

Reply:
that's outdated sorry I extracted it to a variable called `url`